nuget.package/MendixBinaries.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Mendix.LibGit2Sharp.NativeBinaries</id>
+    <version>1.0.1-mx</version>
+    <authors>LibGit2Sharp contributors</authors>
+    <owners>mendix</owners>
+    <licenseUrl>https://raw.githubusercontent.com/libgit2/libgit2/master/COPYING</licenseUrl>
+    <projectUrl>https://github.com/libgit2/libgit2sharp.nativebinaries</projectUrl>
+    <repository url="https://github.com/MarcellVanRooyen/libgit2sharp.nativebinaries.git" type="git"/>
```

Comment:
github.com/mendix ;)

Reply:
updated reference

Reply:
Updated Repository Url to the correct mendix account